package.json

```diff
@@ -11,15 +11,15 @@
     "@gnosis.pm/safe-apps-react-sdk": "3.0.0",
     "@gnosis.pm/safe-deployments": "^1.0.0",
     "@gnosis.pm/safe-react-components": "^0.6.0",
-    "@gnosis/module-factory": "github:gnosis/module-factory#feat/factory-tests",
+    "@gnosis/zodiac": "github:carlosfebres/zodiac#feat-factory-lib",
```

Comment:
One https://github.com/gnosis/zodiac/pull/12 PR is merged, `@gnosis/zodiac` needs to point to master.